test/system/roles/public_health/dashboard/export_verifier.rb

```diff
@@ -177,6 +163,8 @@ def verify_sara_alert_format_export(xlsx, patients)
   end
 
   def verify_full_history_export(xlsx_monitorees, xlsx_assessments, xlsx_lab_results, xlsx_vaccines, xlsx_histories, patients)
+    # Convert patients back into an AR collection for compatability
+    patients = Patient.where(id: patients.id) if patients.is_a?(Patient)
```

Comment:
is it not already an AR collection?

Reply:
I don't believe so in the case of a single patient full export.

Reply:
Just checked. Seems like now it is.